receiver/mysqlreceiver/documentation.md

```diff
@@ -320,6 +320,81 @@ The total time of I/O wait events for a table.
 | table | Table name for event or process. | Any Str |
 | schema | The schema of the object. | Any Str |
 
+### mysql.table.size.avg_row_length
```

Comment:
Is there maybe  a section in the [semconv repo](https://github.com/open-telemetry/semantic-conventions) that covers this?